commands/addons/destroy.js

```diff
@@ -0,0 +1,73 @@
+'use strict'
+
+const cli = require('heroku-cli-util')
+const co = require('co')
+const resolve = require('../../lib/resolve')
+const util = require('../../lib/util')
+const _ = require('lodash')
+
+function * run (context, heroku) {
+  let force = context.flags.force || process.env.HEROKU_FORCE === '1'
+  if (context.args.length === 0) {
+    throw new Error('Missing add-on name')
+  }
+
+  function showRelease (app, attachments) {
+    let names = attachments.map((att) => cli.color.configVar(att.name)).join(', ')
```

Comment:
Not all Attachment-derived config vars will get removed now, see https://github.com/heroku/api/commit/87df3952dbe052e08e289124723b3983b84e1699.

If the User overwrote one with their own value, we don't remove it. The message here might be misleading in that scenario. Not sure how we can fix that in the CLI, short of duplicating the logic in that commit 😒